csclient/csclient_test.go

```diff
@@ -189,9 +186,11 @@ var getTests = []struct {
 }}
 
 func (s *suite) TestGet(c *gc.C) {
-	ch := storetesting.Charms.CharmDir("wordpress")
-	url := newResolvedURL("~charmers/utopic/wordpress-42", 42)
-	err := s.store.AddCharmWithArchive(url, ch)
+	ch := charmRepo.CharmDir("wordpress")
+	err := s.client.UploadCharmWithRevision(
+		charm.MustParseReference("~charmers/utopic/wordpress-42"),
+		ch,
+		42)
```

Comment:
personally I think I'd format this as:
```go
err := s.client.UploadCharmWithRevision(
	charm.MustParseReference("~charmers/utopic/wordpress-42"),
	ch,
	42,
)
```
I like it when the delimiters line up.